app/pdfs/print_record_pdf.rb

```diff
@@ -5,41 +5,44 @@ class PrintRecordPdf < Prawn::Document
   def initialize(record)
     super()
     @record = record
-    page_width = bounds.width
-    column_width = bounds.width / 2
-    header_content(page_width, column_width)
-    table_content(page_width, column_width)
-  end
-
-  def header_content(page_width, column_width)
-    bounding_box([0, cursor], width: page_width) do
-      cell_style = { border_color: 'FFFFFF', align: :center }
-      table text_content, cell_style: cell_style do
-        column(0).font_style = :bold
-        self.column_widths = [column_width, column_width]
-      end
+    stroke do
+      rounded_rectangle [0, bounds.height], bounds.width, bounds.height, 5
     end
-    move_down 10
+    content_width = bounds.width - 10
+    column_width = content_width / 2
+    header_content(content_width)
+    table_content(content_width, column_width)
   end
 
-  def text_content
-    [[@record.user.full_name, format_date_time(@record.created_at)]]
+  def header_content(content_width)
+    date = format_date_time(@record.created_at)
+    name = @record.user.full_name
+    email = @record.user.email
+    bounding_box([5, cursor], width: content_width) do
+      font 'Helvetica'
+      move_down 20
+      text "#{@record.position.name} Application Record", size: 24
+      text "submitted #{date} by #{name}, #{email}"
+      move_down 5
+      move_down 10
+    end
   end
 
-  def table_content(page_width, column_width)
-    bounding_box([0, cursor], width: page_width) do
+  def table_content(content_width, column_width)
+    bounding_box([5, cursor - 5], width: content_width) do
       table data_rows do
-        row(0).font_style = :bold
-        row(0).align = :center
+        style row(0), size: 20, font_style: :bold
+        cells.padding = 12
         self.header = true
-        self.row_colors = %w(DDDDDD FFFFFF)
         self.column_widths = [column_width, column_width]
+        self.cell_style = { borders: [:bottom], border_width: 0.5 }
       end
     end
   end
 
   def data_rows
```

Comment:
While we're here, this kind of data processing seems like it belongs in the model - that way we can test it, and it's not taking up space here.